src/backends/interflop-bitmask/Makefile.am

```diff
@@ -1,9 +1,9 @@
 lib_LTLIBRARIES = libinterflop_bitmask.la
 libinterflop_bitmask_la_SOURCES = interflop_bitmask.c ../../common/logger.c ../../common/options.c
-libinterflop_bitmask_la_CFLAGS = -DBACKEND_HEADER="interflop_bitmask"
+libinterflop_bitmask_la_CFLAGS = -DBACKEND_HEADER="interflop_bitmask" -O3
 if WALL_CFLAGS
 libinterflop_bitmask_la_CFLAGS += -Wall -Wextra
 endif
 libinterflop_bitmask_la_LDFLAGS = -lm
-libinterflop_bitmask_la_LIBADD = 
+libinterflop_bitmask_la_LIBADD = ../../common/rng/libvfc_rng.la
```

Comment:
As a general comment, nothing to be changed yet, but to discuss for later. shall we keep relative path? Or use verificarlo_home variable?

Reply:
I opened an issue to discuss it later but it would make the Makefile cleaner. :+1: